src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/EmailConfiguration.kt

```diff
@@ -177,25 +180,30 @@ fun interface EmailService {
   fun send(email: Email): Result<Pair<UUID, TemplateId>>
 }
 
-abstract class Email(
-  val address: String,
-  prisonerFirstName: String,
-  prisonerLastName: String,
-  date: LocalDate = LocalDate.now(),
-  comments: String? = null,
-) {
-  private val common = mapOf(
-    "date" to date.toMediumFormatStyle(),
-    "prisonerName" to prisonerFirstName.plus(" $prisonerLastName"),
-    "comments" to (comments ?: "None entered"),
-  )
-  private val personalisation: MutableMap<String, String?> = mutableMapOf()
+abstract class Email(val address: String) {
```

Comment:
Needed to abstract out as there was only one email type which was for prisoners.